rqt_py_common/setup.py

```diff
@@ -1,11 +1,8 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Same comment as on [qt_gui_core](https://github.com/ros-visualization/rqt/pull/142/files/3fee97e14908cb061c8977ac5609f1fce965ce32#diff-1ab904ac4fc501b59ea1d8c30ca50ec3).

Reply:
Resolved in #155